From 1a01f748261b5fa27e7ebf5a42751bacaa4f7304 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Mon, 8 Aug 2022 00:36:36 +0200
Subject: Deactivate CONFIG_TOOLS_MKEFICAPSULE

Deactivate mkeficapsule, it is not used by the OpenWrt build system.
mkeficapsule needs gnutls which is not a default dependency of OpenWrt.

Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 configs/tools-only_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/configs/tools-only_defconfig
+++ b/configs/tools-only_defconfig
@@ -34,4 +34,4 @@ CONFIG_I2C_EDID=y
 # CONFIG_VIRTIO_SANDBOX is not set
 # CONFIG_GENERATE_ACPI_TABLE is not set
 # CONFIG_EFI_LOADER is not set
-CONFIG_TOOLS_MKEFICAPSULE=y
+# CONFIG_TOOLS_MKEFICAPSULE is not set
